FinSearchUnified/Helper/StaticHelper.php

```diff
@@ -823,4 +823,30 @@ public static function setPromotion(SimpleXMLElement $xmlResponse)
             ]);
         }
     }
+
+    /**
+     * @param \SimpleXMLElement $xmlResponse
```

Comment:
If the namespace is already imported, make sure to remove the preceding backslash.